lib/ClangImporter/ClangImporter.cpp

```diff
@@ -778,6 +778,12 @@ importer::addCommonInvocationArguments(
     // Enable double wide atomic intrinsics on every x86_64 target.
     // (This is the default on Darwin, but not so on other platforms.)
     invocationArgStrs.push_back("-mcx16");
+    // Enable the FC16/CVT16 extensions which provide half precision floating
+    // point support on x86_64.  This bumps the minimum requirement of the CPU
+    // to ~Haswell, however, Windows 10 (at least as of 1709) requires
```

Comment:
F16C was introduced with Ivybridge, not Haswell.

Reply:
Ivybridge just preceded Haswell on the tick IIRC, but I can absolutely update the comments to be more precise (and I do like precision).  Thanks for the history check @stephentyrone!